packages/manager/modules/pci/src/components/project/storages/databases/engine.class.js

```diff
@@ -33,6 +33,8 @@ export default class Engine {
     });
 
     this.isDefault = some(availability, 'default');
+
+    this.status = availability.filter((a) => a.engine === this.name)[0].status;
```

Comment:
what if filter returns empty array? it will fail in run time.

Reply:
Fixed: https://github.com/ovh/manager/pull/5586/commits/f4c7cf9487f1745c9bc94a262c785528601b64f5